src/components/LogViewer/LogViewerToolbar.tsx

```diff
@@ -0,0 +1,95 @@
+import { Button, Checkbox, Toolbar, ToolbarContent, ToolbarGroup, ToolbarItem } from '@patternfly/react-core';
+import { LongArrowAltDownIcon, LongArrowAltUpIcon } from '@patternfly/react-icons';
+import { Fragment, ReactNode } from 'react';
+
+import { TooltipWrapper } from 'components/TooltipWrapper/TooltipWrapper';
+
+interface ILogViewerToolbarProps {
+  logViewerRef: React.RefObject<any>;
+  isStatic: boolean;
+  setIsPaused: (value: boolean) => void;
+  isFollowing: boolean;
+  setIsFollowing: (value: boolean) => void;
+  areLinesWrapped: boolean;
+  setAreLinesWrapped: (value: boolean) => void;
+  customActions?: ReactNode[];
+}
+
+export const LogViewerToolbar = ({
```

Comment:
Maybe consider using `React.memo` for child component, but I will leave it to you to decide whether to use memo or not.